config/routes.rb

```diff
@@ -5,7 +5,13 @@
     root to: 'home#index'
   end
 
-  devise_for :users, only: [:sessions]
+  devise_for :users, only: [:sessions, :authy], :path_names => {
+    :verify_authy => "/verify-token",
```

Comment:
prefer Ruby 1.9+ hash syntax instead of fat arrow, e.g. `verify_authy: "/verify-token",`